draft-ietf-ccwg-bbr.md

```diff
@@ -293,8 +293,26 @@ e.g., via a QUIC ACK Range {{RFC9000}}, TCP cumulative acknowledgment
 
 ## Transport Connection State {#transport-connection-state}
 
-C.SMSS: The Sender Maximum Segment Size in bytes. This is similar to PMTU
-in QUIC {{RFC9000, Section 14.2}}.
+C.SMSS: The Sender Maximum Send Size in bytes. The maximum
+size of a single packet transmission, including the portion
```

Comment:
Isn't this identical to the portion of the packet the transport tracks for congestion control purposes?

```suggestion
size of a single transmission, including the portion
```